### PR TITLE
Prune command

### PR DESCRIPTION
**Pull request by someaddons**
# Changes proposed in this pull request:
Adds a server utility prune world command, which will remove regions files which arent within a given range of a colony building. Useful to prune existing worlds, but keeping important colony places intact.

Review please

### CODE REVIEW

**Comment by Raycoms** [comment]
Looks sound :shipit: 

[Approved with PullApprove](https://pullapprove.com/ldtteam/minecolonies/pull-request/6114/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ldtteam/minecolonies)

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 1]
```diff
@@ -430,6 +430,8 @@
   "com.minecolonies.command.playernotfound": "Player %s could not be found.",
   "com.minecolonies.command.notenabledinconfig": "This command is disabled in the config.",
   "com.minecolonies.command.resetsupply": "Successfully reset the Supply Camp and Ship placement.",
+  "com.minecolonies.command.prune.next": "This command will delete large chunks of your world! Use with care! Make a backup first! If you want to continue repeat this command with %d as first parameter",
```
Comment (ravenbuilder934) [inline src/main/resources/assets/minecolonies/lang/en_us.json]:
```suggestion
  "com.minecolonies.command.prune.next": "This command will delete large chunks of your world! Use with care! Make a backup first! If you want to continue, repeat this command with %d as the first parameter.",
```

**Review by ravenbuilder934 — CHANGES_REQUESTED** [review]

**Review by Mekle001 — APPROVED** [review]
This looks good.

**Comment by Mekle001** [comment]
One thing that might be interesting in the future: A version of this that only works in the current world, or a specified world. I can see this being useful in many scenarios to only prune a single world.

**Comment by someaddons** [comment]
> One thing that might be interesting in the future: A version of this that only works in the current world, or a specified world. I can see this being useful in many scenarios to only prune a single world.

ye I've been thinking about that aswell

**Review by marchermans — APPROVED** [review]

**Pull request merged**